### PR TITLE
v1.3.1

### PR DESCRIPTION
**Pull request by DAB0mB**

### CODE REVIEW

File: scripts/bundle-meteor-base-min.sh [hunk 1]
```diff
@@ -37,13 +37,17 @@ mkdir $OUTPUT_PATH
 
 # Concat files
 cat "$PACKAGES_PATH/underscore.js" >> $OUTPUT_PATH/$NAME.bundle.js
+cat "$PROJECT_ROOT/meteor-runtime-config.js" >> $OUTPUT_PATH/$NAME.bundle.js
```
Comment (idanwe) [inline scripts/bundle-meteor-base-min.sh:40]: 👍

**Comment by idanwe** [comment]
@DAB0mB thank you

**Comment by telemakhos** [comment]
@idanwe Is this Meteor 1.3.1 support?

**Comment by idanwe** [comment]
Yes

**Pull request merged**